settings.gradle.kts

```diff
@@ -443,6 +443,6 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = "feeder"
+rootProject.name = "Feeder"
```

Comment:
This doesn't seem relevant to this PR.

Rename your directory to `feeder` instead?

Maybe I can rename the repo or something..

Reply:
Renaming the folder to lowercase didn't work. The app won't sync for me without it. will drop for this change, but I need to apply it for myself to build the app

![image](https://github.com/spacecowboy/Feeder/assets/171704/66254092-c62c-4a9f-87fc-ffc84ac2ef0f))